ports/libsimpleservo/api/Cargo.toml

```diff
@@ -7,6 +7,7 @@ edition = "2018"
 publish = false
 
 [dependencies]
+clipboard = "0.5"
```

Comment:
Since libsimpleservo/api is a platform-agnostic embedding API, we should not add a platform-specific dependency like clipboard to it.